aspnetcore/signalr/authn-and-authz/sample/wwwroot/js/chat.js

```diff
@@ -0,0 +1,142 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
Add the generated *.js files to a *.gitignore file.

Reply:
Yeah, I can do that here. Generally we've tried to ensure there were JavaScript versions of the samples but that's difficult in this case since it's a much more complicated sample.